packages/blade/scripts/generateCssThemeVariables.js

```diff
@@ -0,0 +1,196 @@
+const { getLeaves } = require('any-leaf');
+const StyleDictionary = require('style-dictionary');
+const set = require('lodash/set');
+const cloneDeep = require('lodash/cloneDeep');
+const chalk = require('chalk');
+const figures = require('figures');
+
+const { paymentTheme, bankingTheme } = require('../build/js-bundle-for-css/tokensBundle');
+const {
+  makeBorderSize,
+  makeMotionTime,
+  makeSpace,
+  makeTypographySize,
+  makeSize,
+} = require('../build/js-bundle-for-css/utilsBundle');
+
+const getThemeFromTokens = ({ onColorMode, onDeviceType, themeTokens }) => {
+  return {
+    ...themeTokens,
+    colors: themeTokens.colors[onColorMode],
+    shadows: {
+      ...themeTokens.shadows,
+      color: themeTokens.shadows.color[onColorMode],
+    },
+    typography: themeTokens.typography[onDeviceType],
+  };
+};
+
+const getStyledDictionaryConfig = ({ outputFileName, tokens }) => {
+  return {
+    tokens,
+    platforms: {
+      css: {
+        transformGroup: 'css',
+        buildPath: 'build/css/',
+        files: [
+          {
+            destination: `${outputFileName}.css`,
+            format: 'css/variables',
+          },
+        ],
+      },
+    },
+  };
+};
+
+const makeValue = ({ path, value }) => {
```

Comment:
We're doing some fuzzy checks here with `.includes`. Can this cause any issues considering there are ton of css properties? For example - say we've border.color, border.radius -> both pass for the first check 🤔

Reply:
Yeah, it could cause issues. We will have to put some kind of checks somewhere in order to parse it. Fuzzy checks seem much easier at the moment, if we see more regressions coming in the future we can add more strict checks for parsing values.

Reply:
I'll fix the border issue now